app/models/user.rb

```diff
@@ -1,9 +1,12 @@
 class User < ApplicationRecord
 
+  has_many :created_tests, :class_name => "Test", inverse_of: 'author'
```

Comment:
```suggestion
  has_many :created_tests, class_name: "Test", inverse_of: 'author', foreign_key: 'user_id'
```
насколько я помню доки, рекомендуется (при несовпадении имени ассоциации и имени внешнего ключа со стороны "зависимого объекта") опцию `foreign_key:` использовать с обеих сторон ассоциации